05-lists-and-dictionaries/capital_city_loop.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2016 Ami . All rights reserved
+
+from capitals import capitals_dict
+from random import choice
+
+def get_random_state():
+    return choice(list(capitals_dict.keys()))
+
+def capital_of_state_quiz(state):
+    while True:
+        response = input("What is the capital of " + state + "? ").lower()
+
+        if check_answer(response, capitals_dict[state]):
+            print("Correct")
+            return True
```

Comment:
I missed this earlier, you can just return here (you don't return a value to the `while` but to the caller of the function, which in this case doesn't use the value so no need to return any value.